public/local/live_config.html

```diff
@@ -16,6 +16,12 @@ <h2>Hello broadcaster!</h2>
         <div>
             <input type="text" id="input" disabled="disabled" placeholder="Enter a question"/>
         </div>
+        <div>
+            <input type="text" id="optionA" disabled placeholder="Enter option A"
```

Comment:
does the 'disabled' attribute work this way? or is it necessary to use 'disabled = "disabled"'?